test/config.js

```diff
@@ -392,6 +403,27 @@ exports.merged = {
     },
 }
 
+exports.getInt = {
+    'setUp' : setUp,
+    // config.get('name');
+    'test (non-existing)' : function (test) {
+        test.expect(2);
+        const result = this.config.getInt('test-non-exist');
+        test.equal(typeof result, 'number');
+        test.ok(isNaN(result));
+        test.done();
+    },
+    'test non-existing with default' : function (test) {
+        _test_int(test, 'test-non-exist', 5, 5);
+    },
+    'test.int equals 6' : function (test) {
+        _test_int(test, 'test.int', undefined, 6);
+    },
+    'test.int equals 6 (with default 7)' : function (test) {
+        _test_int(test, 'test.int', 7, 6);
+    },
```

Comment:
Given my comment above, another test should be a non-existing config should be falsy.

Reply:
Good call, updated accordingly.